docs/client/cli/kyuubi_beeline.md

```diff
@@ -0,0 +1,77 @@
+
+- `hive-site.xml`: It is one of the main configuration files for Apache Hive. It contains global configuration properties in Hive services like HS2 hostname and port information, Kerberos configuration properties, SSL properties, etc.
+
+## How users automatically connect to Kyuubi Servers via properties files
+
+Kyuubi Beeline looks for all configuration files sequentially once it gets started.
+
+1. `beeline-site.xml`: Kyuubi Beeline starts by retrieving the named URL from this file.
+
+2. `beeline-hs2-connection.xml`: This file is checked next, and any properties in this file take precedence over those in `beeline-site.xml`.
+
+3. `kyuubi-defaults.conf`: Properties from this file are loaded next. Values in this file may merge with the values of hiveconf from `beeline-hs2-connection.xml`.
+
+4. `hive-site.xml`: Any remaining connection properties are picked up from this file. If properties are present in multiple configuration files, the value from other files (e.g. `beeline-site.xml`, `beeline-hs2-connection.xml`, and `kyuubi-defaults.conf`) takes priority over `hive-site.xml`.
+
+Finally, Kyuubi Beeline generates the complete JDBC connection URL to be used when connecting to Kyuubi servers.
+
+### Tips
+
+Some settings may not be supported by the Kyuubi Beeline client due to compatibility issues.
+The configurations defined in `kyuubi-defaults.conf` might break compatibility with Kyuubi Beeline.
+Therefore, it is recommended to verify the Kyuubi configuration properties before starting Kyuubi Beeline to avoid potential issues.
+
+| Key                                                    | Comment                                           |
+|--------------------------------------------------------|---------------------------------------------------|
+| kyuubi.authentication.ldap.baseDN                      | comma-separated values are not supported          |
+| kyuubi.authentication.ldap.binddn                      | comma-separated values are not supported          |
+| kyuubi.authentication.ldap.customLDAPQuery             | comma-separated values are not supported          |
+| kyuubi.authentication.ldap.groupDNPattern              | comma-separated values are not supported          |
+| kyuubi.authentication.ldap.groupFilter                 | comma-separated values are not supported          |
+| kyuubi.authentication.ldap.userDNPattern               | comma-separated values are not supported          |
+| kyuubi.authentication.ldap.userFilter                  | comma-separated values are not supported          |
+| kyuubi.batch.conf.ignore.list                          | comma-separated values are not supported          |
+| kyuubi.frontend.protocols                              | comma-separated values are not supported          |
+| kyuubi.frontend.thrift.binary.ssl.disallowed.protocols | The value(REST) is an existing keyword in Beeline |
```

Comment:
let's use word "Kyuubi Beeline" to distinguish with Hive Beeline
```suggestion
| kyuubi.frontend.thrift.binary.ssl.disallowed.protocols | The value(REST) is an existing keyword in Kyuubi Beeline |
```

Reply:
Got it. Thanks.